docs/proposals/failed-volume-proposal.md

```diff
@@ -0,0 +1,20 @@
+# Proposal: volume statuses
+
+Last updated: 01.02.22
+
+## Background
+
+Volume has the terminated state which is called "Failed". 
+This state can be reached by some reason, but there is no way to go out from this state.
+Also, it needs to remove Failed volume in the most cases. 
+So, this proposal should suggest a work flow to remove Failed volume and its additional statuses, 
```

Comment:
Remove failed volume or related custom resource?

Reply:
It looks everything k8s works with is a CR.